doc/texture_packs.txt

```diff
@@ -168,3 +168,46 @@ Here are face selectors you can choose from:
 | bottom        | y-                                                |
 | sides         | x-, x+, z-, z+                                    |
 | all           | All faces. You can also use '*' instead of 'all'. |
+
+Designing leaves textures for the leaves rendering options
+----------------------------------------------------------
+
+MT has 3 ways of rendering leaf textures if the leaf has the `allfaces_optional`
+drawtype.
+
+### Fancy
+
+Uses the texture specified in the `tiles` nodedef field.
+The texture should have some transparent pixels and be in the RGBA form so that
+the transparent pixels can have color information.
+Faces of every leaf node are rendered even if they are part of a solid volume of
+leaves, this gives a dense appearence.
+
+### Opaque
+
+Uses the texture specified in `tiles` but makes it opaque by converting each
+transparent pixel into an opaque pixel that use the color information of that
+transparent pixel.
+Due to this the `tiles` texture must be RGBA not 'indexed alpha' to allow each
+transparent pixel to have individual color information.
+
+The colors of the transparent pixels should be set for a good appearence in
+`opaque` mode. This can be done by painting the pixels the desired colors then
+erasing them. Then when saving the texture using the 'save color information
```

Comment:
Improvement (because it makes it a full sentence):  "Then when saving the texture, use the 'save color information ..."